tests/foreman/ui/test_syncplan.py

```diff
@@ -83,14 +39,29 @@ def test_positive_create_basic(self, test_data):
 
         """
         with Session(self.browser) as session:
-            make_syncplan(
-                session,
-                org=self.organization.name,
-                name=test_data['name'],
-                description=test_data['desc'],
-                sync_interval=test_data['interval'],
-            )
-            self.assertIsNotNone(self.syncplan.search(test_data['name']))
+            for value in generate_strings_list():
```

Comment:
@oshtaier Note that previously name/desc/interval were being tested together in one single test case.  IMO this is not the best way.  Now I have split the tests into two - one testing name/desc and other testing the sync intervals.  So the coverage is not reduced.

Also I named the variable 'value' so I can reuse the same variable for both name and desc.